docs/regen-ledger/interfaces.md

```diff
@@ -0,0 +1,39 @@
+# Regen Ledger API & CLI
+
+Regen Ledger provides three interfaces for interacting with a node:
+
+- command-line interface
+- gRPC interface
+- REST interface
+
+Each interface can be used to query the state. The command-line interface can be used to generate, sign, and broadcast transactions, whereas the gRPC and REST interfaces can only be used to broadcast transactions (a transaction needs to be generated and signed either programmatically or using the command-line interface before it can be broadcasted using the gRPC or REST interfaces).
+
+## Command-Line Interface
+
+The most straightforward way to interact with a node is using the command-line interface.
+
+The `regen` binary serves as the node client and the application client, meaning the `regen` binary can be used to both run a node and interact with it. In [Quick Start](../getting-started/), we started a local node using the `regen` binary and then interacted with that node by submitting queries and transactions. For more examples of interacting with a node using the command-line interface, see [Tutorials](../tutorials/).
+
+## gRPC Interface
+
+[gRPC](https://grpc.io/docs/what-is-grpc/introduction/) is a modern RPC framework that leverages [protocol buffers](https://developers.google.com/protocol-buffers) for encoding requests and responses between a client and service. In the case of Regen Ledger, we use gRPC mostly for querying blockchain state (credit or token balances, data signature records, etc). As a client developer, this means that you can query Regen Ledger state directly by using a gRPC library for your language, in combination with Regen Ledger's protobuf definitions defined [here](https://github.com/regen-network/regen-ledger/tree/master/proto/regen).
```

Comment:
I think it'd still be useful to link `grpcurl` as a resource and maybe give a 1-liner of how to run the query commands and see services?

Reply:
Added an example and link to the SDK documentation for each section. :+1:

Reply:
doh! I only saw what you added above. I see the one below now. thx!